### ISSUE TITLE
Weird pixelation issue

### ISSUE THREAD

**Issue by yaSebastian** [brndd/vanilla-tweaks#35]
>[!note]
**TL;DR:** This behavior seems to be caused by the .exe file name and not by the tweaking process.

# Details

## Symptoms

I just ran into some weird pixelation behavior.
The tweaked executable gave me this kind of pixelation everywhere in the game, labels, quest texts, toolbars, settings, etc., while the rendering of the 3D world seems to be unaffected.
Example:
<img width="228" alt="image" src="https://github.com/brndd/vanilla-tweaks/assets/17569304/22985990-12e2-411e-8d35-3ee8aea126eb">

The original wow.exe looks normal, as far as I can tell.
Example:
<img width="227" alt="image" src="https://github.com/brndd/vanilla-tweaks/assets/17569304/2a7306c6-02f2-4b19-b7ff-3d598370f290">

## Analysis

I first attributed this behavior to the vanilla-tweaks tweaking process. As I couldn't find anything about it neither in the issues of this repository nor on google in general, I tried to pinpoint what single change might be the source of it. By chance, I realized (for whatever weird reason), the behavior is tied to the file name of the .exe itself.
It affects the original executable just the same way it does affect the tweaked ones. Whenever the file name is is wow.exe (casing seems to be irrelevant), I get the regular rendering, whenever it differs, I get the pixelated rendering. (Different compatibility settings on the .exe's didn't seem to affect the behavior either.)

## Context

I am running the Windows enUS 1.12.1.5875 Client.
Operating System is a Windows 11 virtual machine running in the more recent version of Parallels on a M2 Ultra Mac with the most current macOS.
The virtual machine has been installed just the other day, so I haven't made many changes to any configuration. The VM has just been set to game mode and Windows has been set to 200% DPI. Also, Windows has not been activated yet.

## Issue justification

Even though I'm pretty sure now this is in no way related to vanilla-tweaks, others might run into the same behavior, as the new executable gets a new name by default. I hope this summary might save someone else some time when they run into the same behavior.

@brndd , feel free to close this issue right away. Given you have some experience with the game engine, if you know the source of this behavior, I'd be very curious about a short explanation. Thanks for this awesome project!

**Comment by brndd** [comment]
I'm guessing it's a video driver thing. Drivers often have game-specific behaviour that depends on the name of the process. Just one more reason these hacks would be better as a drop-in DLL mod, I suppose.

**Comment by yaSebastian** [comment]
Thanks for the reply. The drop-in DLL might be a better approach, but I guess it would be more complex and time consuming. For me, once you know the source of the pixelation issue, renaming the .exes works well enough.
